core/statistics/resourceMonitor.go

```diff
@@ -76,3 +97,13 @@ func (rm *ResourceMonitor) Close() error {
 	rm.file = nil
 	return err
 }
+
+func getCurrentProcess() (*process.Process, error) {
```

Comment:
This function is now in statistics/machine/common.go and you should move file common.go in statistics folder to can access function getCurrentProcess() also in machine package and statistics package.

Reply:
Done. If I move common.go in statistics, I won't have access to the function from the machine subpackage, so I exported the func.